### PR TITLE
Update readme and translations

### PR DESCRIPTION
**Pull request by Johnmacrocraft**
### Description

This pull request updates readme and translations.

### Reason to modify

So, this pull request won't break things, and there was typo in readme and also word _**Tesseract**_ from [Tesseract](https://github.com/TesseractTeam/Tesseract) in translations, I decided to create this pull request.

### Tests & Reviews

Unfortunately I haven't tested the code, but it doesn't seem to have any errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +8,7 @@ Introduction
 -------------
 Genisys 3 is one of several derived projects resulting from the inactivity of **[PocketMine/PocketMine-MP](https://github.com/PocketMine/PocketMine-MP)** & **[ITX Tech](https://github.com/iTXTech/Genisys)** which is now commonly known as **[pmmp/PocketMine-MP](https://github.com/pmmp/PocketMine-MP)**. It has many gameplay features implemented which PocketMine-MP does not since Minecraft PE v0.11.
 
-[Genisys](https://github.com/iTXTech/Genisys/) has officially died and [FrontierDevs](https://github.com/FrontierDevs) are to revive it under the name Genisys3 
+[Genisys](https://github.com/iTXTech/Genisys/) has officially died and [FrontierDevs](https://github.com/FrontierDevs) are trying to revive it under the name Genisys3
```
Comment (InspectorGadget) [inline README.md]: We aren't trying

Comment (Johnmacrocraft) [reply README.md]: @RTGDaCoder
[Genisys](https://github.com/iTXTech/Genisys/) has officially died and [FrontierDevs](https://github.com/FrontierDevs) are reviving it under the name Genisys3

Emm than, what about this? Is it okay?

Comment (InspectorGadget) [reply README.md]: Yes

Comment (Johnmacrocraft) [reply README.md]: @RTGDaCoder Done, I changed it.

**Review by InspectorGadget — APPROVED** [review]

**Comment by InspectorGadget** [comment]
Merged, thanks :)

**Pull request merged**